presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -331,11 +340,13 @@ private PlanNodeStatsEstimate crossJoinStats(JoinNode node, PlanNodeStatsEstimat
     {
         private final PlanNodeStatsEstimate stats;
         private final Optional<Symbol> selectedClauseSymbol;
+        private final double selectedClauseSymbolNdv;
 
-        public AntiJoinStats(PlanNodeStatsEstimate stats, Optional<Symbol> selectedClauseSymbol)
+        public AntiJoinStats(PlanNodeStatsEstimate stats, Optional<Symbol> selectedClauseSymbol, double selectedClauseSymbolNdv)
```

Comment:
`selectdClauseSymbolNdv` is required in order to cap NDVs when adding inner join to anti join stats. It is now possible that inner NDVs + anti join NDVs would be larger than original column NDVs.
I will probably just pass max NDV explicitly